### PR TITLE
Add time entry suggestion cell

### PR DESCRIPTION
**Pull request by willsb**
Similar to #417
Equally blocking

Squash whenever.

### CODE REVIEW

File: Toggl.Foundation.MvvmCross/ViewModels/TimeEntrySuggestionViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+﻿using MvvmCross.Core.ViewModels;
+using Toggl.Foundation.MvvmCross.Helper;
```
Comment (arturs105) [inline Toggl.Foundation.MvvmCross/ViewModels/TimeEntrySuggestionViewModel.cs]: Unnecessary

File: Toggl.Foundation.MvvmCross/ViewModels/ProjectSuggestionViewModel.cs [hunk 2]
```diff
@@ -13,11 +15,11 @@ public sealed class ProjectSuggestionViewModel : MvxNotifyPropertyChanged, ITime
 
         public string ProjectColor { get; }
 
-        public ProjectSuggestionViewModel(IDatabaseProject project, int numberOfTasks)
+        public ProjectSuggestionViewModel(IDatabaseProject project)
```
Comment (arturs105) [inline Toggl.Foundation.MvvmCross/ViewModels/ProjectSuggestionViewModel.cs]: Why was number of tasks removed?

File: Toggl.Daneel/Views/StartTimeEntryViewCell.cs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        protected StartTimeEntryViewCell(IntPtr handle) : base(handle)
+        {
+            // Note: this .ctor should not contain any initialization logic.
+        }
+
+        public override void AwakeFromNib()
+        {
+            base.AwakeFromNib();
+
+            ContentView.BackgroundColor = UIColor.White;
```
Comment (arturs105) [inline Toggl.Daneel/Views/StartTimeEntryViewCell.cs]: Can't this be set in the interface builder?

File: Toggl.Foundation.MvvmCross/ViewModels/ProjectSuggestionViewModel.cs (hunk 2 above)
Comment (willsb) [inline Toggl.Foundation.MvvmCross/ViewModels/ProjectSuggestionViewModel.cs]: We are not fetching tasks anytime soon, might as well only add this when needed

**Pull request merged**